srcpkgs/intermodal/template

```diff
@@ -1,11 +1,28 @@
 # Template file for 'intermodal'
 pkgname=intermodal
-version=0.1.10
+version=0.1.12
 revision=1
 build_style=cargo
+build_helper=qemu
+make_check_args="-- --skip subcommand::torrent::verify::tests::output_multiple --skip subcommand::torrent::verify::tests::output_color"
 short_desc="User-friendly and featureful command-line BitTorrent metainfo utility"
 maintainer="Pika <pika@lasagna.dev>"
 license="CC0-1.0"
 homepage="https://github.com/casey/intermodal"
 distfiles="https://github.com/casey/intermodal/archive/v${version}.tar.gz"
-checksum=8bb3e4f549e5c4446543babd741fec0cd7c42da4d49eca3e98c5f7611ad59618
+checksum=cd62894e519dc5aa0284a5f48aab86e1a45c3bc96b8a5481741adb6960d4751a
+
+post_build() {
+	IMDL="target/${RUST_TARGET}/release/imdl"
+	ln -sf "${IMDL}" imdl
+
+	for shell in zsh bash fish; do
+		vtargetrun ./imdl completions ${shell} >imdl.${shell}
```

Comment:
...because you can just run"$IMDL" right here.

Reply:
cargo-culted from our rustup template, but yeah, seems a bit redundant to have both the variable *and* the link